Patterns/Korean/Korean-Numbers.yaml

```diff
@@ -40,15 +38,6 @@ ZeroToNineMap: !dictionary
     팔: 8
     구: 9
     빵: 0
-    하나: 1
```

Comment:
Are these really incorrect values? Or will they come back in future refinements of the code?

Reply:
In Korean, "하나둘~~~아홉" used to indicate the number of objects rather than "Number 1". 

To represent a simple number, we use "일이삼사오육칠팔구" rather than "하나둘~~~아홉"

So I removed that expression in yaml.